.env.example

```diff
@@ -1,16 +1,16 @@
 APP_NAME=Laravel
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:ESNs7F5DT76xopqmdq4sOTa6p8jVALNiMq09fYLdR6c=
 APP_DEBUG=true
 APP_URL=http://localhost
 
 LOG_CHANNEL=stack
 
-DB_CONNECTION=mysql
+DB_CONNECTION=pgsql
```

Comment:
Apenas uma observação: acredito que não seja necessário alterar esse arquivo, mas entendo que deve ter sido alterado para facilitar o início do projeto ao realizar o `git clone`.